src/api/component.js

```diff
@@ -72,13 +72,12 @@ export default class Component extends HTMLElement {
       // then that causes the attribute to be set again.
       if (propData.syncingAttribute) {
         propData.syncingAttribute = false;
-        return;
+      } else {
+        // Sync up the property.
+        const propOpts = this.constructor.props[propertyName];
+        propData.settingAttribute = true;
```

Comment:
i am probably wrong, but should this be `propData.syncingAttribute`? just asking because of the if condition